runtimes/onnx-tf/development/Dockerfile

```
@@ -27,7 +27,8 @@ RUN pip3 install onnx
 
 # Install dependencies
 RUN pip3 install --no-cache-dir \
-    tensorflow
+    tensorflow \
+    tensorflow-probability
 RUN pip3 install git+https://github.com/onnx/onnx-tensorflow.git@master
```

We still need to update the branch name from `master` -> `main`
```suggestion
RUN pip3 install git+https://github.com/onnx/onnx-tensorflow.git@main
```